examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -9,9 +9,7 @@
 // ignore_for_file: unchecked_use_of_nullable_value, unnecessary_library_directive
```

Comment:
I'm not as familiar with the dart.dev codebase, but why is this examples directory at the root as opposed to src/?

Reply:
It's the same as `flutter/website`, but I guess I don't have a reason. They've just always been that way since I started working on the sites.

Perhaps it makes sense though, as the content isn't actually used in the site or by the site generation itself, but rather by the separate code excerpt and testing functionality.

Happy to discuss the examples directory more if you're interested or have any thoughts! :D

Reply:
This seems to be out of scope for this project haha but worth discussing later on once migration is done!